guide/content/en/guide/basics/sync_async.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Sync and Async in Sanic
+---
+
+# Sync and Async in Sanic
+
+Sanic is an **asynchronous** framework - it is designed to be responsive to requests, which allows it to theoretically serve more requests, more quickly, than a synchronous framework. Other python asynchronous frameworks also share the same limitations when calling synchronous code, so the guide below will hopefully serve you if you end up in a situation using something other than Sanic in the future.
```

Comment:
```suggestion
Sanic is an **asynchronous** framework—it is designed to be responsive to requests, which allows it to theoretically serve more requests, more quickly, than a synchronous framework. Other python asynchronous frameworks also share the same limitations when calling synchronous code, so the guide below will hopefully serve you if you end up in a situation using something other than Sanic in the future.
```

Reply:
Sorry, the typographer in me cannot overlook the mistake of a hyphen for a dash.